src/main/java/gyro/google/compute/BackendBucketCdnPolicy.java

```diff
@@ -60,6 +72,154 @@ public void setSignedUrlKeyNames(List<String> signedUrlKeyNames) {
         this.signedUrlKeyNames = signedUrlKeyNames;
     }
 
+    /**
+     * Specifies the cache mode for all responses from this backend bucket.
+     */
+    @ValidStrings({"CACHE_ALL_STATIC", "FORCE_CACHE_ALL", "INVALID_CACHE_MODE", "USE_ORIGIN_HEADERS"})
+    @Required
```

Comment:
Should this be a required field? We've been using this resource without the `cacheMode` and it doesn't seem like it has been an issue.